src/server/pps/server/api_server.go

```diff
@@ -1273,6 +1277,25 @@ func setDefaultJobInputMethod(inputs []*ppsclient.JobInput) {
 	}
 }
 
+const (
+	defaultCPU = 0.25
+	defaultMem = "250M"
+)
+
+// maybeDefaultResouceRequest returns 'res' if it's not nil, and a default
+// resource request it if is. This makes sure that pipeline/job workers are
+// given a baseline set of compute resources if none are specified in the
+// request.
+func maybeDefaultResourceRequest(res *ppsclient.Resources) *ppsclient.Resources {
```

Comment:
Personally I think this name is confusing ... I'd just name it `getResourceRequest` and its ok if it includes default behavior. But that's just me. This is a nitpick, not a dealbreaker.